bin/compile

```diff
@@ -72,6 +72,10 @@ pdo_mysql.default_socket = /var/run/mysqld/mysqld.sock
 xdebug.max_nesting_level = 256
 EOF
 
+cat >> "${INSTALL_DEST}/${VERSION}/etc/conf.d/pcov.ini" <<EOF
```

Comment:
This doesn't look right to me with fresh eyes this morning, this wants to be written after the extension is installed, but I'm not sure what is the best way to do that ... maybe just write this line into the global ini ?